### ISSUE TITLE
Error with role Tag in HTML

### ISSUE THREAD

**Issue by ingeniumdesign** [dmind-gmbh/extension-cookieman#338]
# Bug Report

## Prerequisites

* [x] Can you reproduce the problem on TYPO3 master

## Description

in the Google Website Audit Tool:
https://pagespeed.web.dev/analysis/

```
Elements with `role="dialog"` or `role="alertdialog"` do not have accessible names.
ARIA dialog elements without accessible names may prevent screen readers users from discerning the purpose of these elements.
```

```
Elemente mit role="dialog" oder role="alertdialog" haben keinen barrierefreien Namen.
ARIA-Dialogelemente ohne barrierefreie Namen können Nutzer von Screenreadern daran hindern, den Zweck dieser Elemente zu erkennen. Informationen zum Verbessern der Barrierefreiheit von ARIA-Dialogelementen
Fehlerhafte Elemente
Hinweis zu Cookies Unsere Webseite verwendet Cookies. Diese haben zwei Funktio…
<div id="cookieman-modal" data-cookieman-showonce="1" data-cookieman-settings="{&quot;groups&quot;:{&quot;mandatory&quot;:{&quot;preselected&quot;:true,&quot;disabled&quot;:true,&quot;trackingObject…" class="modal fade show" tabindex="-1" role="dialog" aria-modal="true" style="display: block;">
```

## Steps to Reproduce

https://dequeuniversity.com/rules/axe/4.9/aria-dialog-name

## Versions

T3v12.x

**Comment by jonaseberle** [comment]
Hi, thanks for reviewing.
Please be more specific.
You mean `aria-title` is missing on the dialog modals?
I assume you mean the `bootstrap5-modal` default theme?

**Comment by jonaseberle** [comment]
I added `aria-title="Cookie Dialog"`
